src/OpenTelemetry.ResourceDetectors.Process/README.md

```diff
@@ -0,0 +1,39 @@
+# Process Runtime Resource Detectors
+
+[![NuGet version badge](https://img.shields.io/nuget/v/OpenTelemetry.ResourceDetectors.Process)](https://www.nuget.org/packages/OpenTelemetry.ResourceDetectors.Process)
+[![NuGet download count badge](https://img.shields.io/nuget/dt/OpenTelemetry.ResourceDetectors.Process)](https://www.nuget.org/packages/OpenTelemetry.ResourceDetectors.Process)
+
+## Getting Started
+
+You need to install the
+`OpenTelemetry.ResourceDetectors.Process` package to be able to use the
+Process Runtime Resource Detectors.
+
+```shell
+dotnet add package OpenTelemetry.ResourceDetectors.Process --prerelease
+```
+
+## Usage
+
+You can configure Process Runtime resource detector to
```

Comment:
Please mention that Resource can be applied to all signals, and if feasible show them each here. Or mention that "while Resource can be applied to all signals, this particular snippet is showing how to add for tracing signal".

Reply:
#1509